packages/nimble-components/src/banner/styles.ts

```diff
@@ -0,0 +1,252 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    BannerFail100DarkUi,
+    Black75,
+    Fail100LightUi,
+    Information100DarkUi,
+    Information100LightUi,
+    Warning100DarkUi,
+    Warning100LightUi,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+
+import {
+    actionRgbPartialColor,
+    bodyFont,
+    borderHoverColor,
+    buttonLabelFontColor,
+    controlHeight,
+    fillSelectedColor,
+    iconColor,
+    iconSize,
+    linkActiveFontColor,
+    linkDisabledFontColor,
+    linkFontColor,
+    slimControlHeight
+} from '../theme-provider/design-tokens';
+import { Theme } from '../theme-provider/types';
+import { hexToRgbaCssColor } from '../utilities/style/colors';
+import { MultivaluePropertyStyleSheetBehavior } from '../utilities/style/multivalue-property-stylesheet-behavior';
+import { themeBehavior } from '../utilities/style/theme';
+import { BannerSeverity } from './types';
+
+export const styles = css`
```

Comment:
Feedback from demo: add some padding between the text content and the action slot

Reply:
Added